text/0265-event-vision.md

```diff
@@ -0,0 +1,74 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to generic logs.
+
+Because of this, the OpenTelemetry project wants to encourage the transition from generic logs to events.
+
+### OTLP
+
+Since events are a type of log, they share the same OTLP data structure and OTLP pipeline.
+
+### API
+
+OpenTelemetry should have an Event API. This will help to promote the distinction between events and generic logs,
+and encourage the use of events over generic logs.
+
+### Interoperability with generic logging libraries
```

Comment:
TODO how does event body interop with generic logging libraries? (including anyvalue bodies)

possibility: map event body to jsonified string "generic log" body

Reply:
I suggest that in the generic case we follow the same rules we have for mapping event attributes and the event body to a span event.

Custom mappings for event features will be a common request. These requests are easily handled by event processors and the log bridge, as it is simple to write tiny plugins that do these conversions.

Reply:
(btw, since this OTEP is meant to be a framing document, I suggest that we avoid defining the specifics for how Events and SpanEvents map to each other.)